### DISCUSSION TITLE
Issues with SNZB-04 and MOSZB-140

### DISCUSSION THREAD

**Discussion by federicosantolini** [Koenkk/zigbee2mqtt#14630] [Devices]
Ciao everyone!
I have an issue with some Sonoff **SNZB-04** and Frient **MOSZB-140**.

The Frient ones can be paired and they're recognized but I miss almost every sensor. I really don't understand why.
![image](https://user-images.githubusercontent.com/94896891/197528834-0dca9df6-6886-4317-842f-489775738795.png)
![image](https://user-images.githubusercontent.com/94896891/197528884-2c48da4a-88a9-4a12-b23f-9fec7a92c970.png)

The Sonoff door sensors are not recognized at all, cannot complete the interview but [I see that this device is fully compatible](https://www.zigbee2mqtt.io/devices/SNZB-04.html).
![image](https://user-images.githubusercontent.com/94896891/197529343-1bf7daee-9496-4446-baee-7e12e30a9f3d.png)

I tested the sonoff with a friend's installation where he has Z2M with a sonoff zigbee antenna. (I have the conbee one but all of these devices were working in ZHA)

If someone can please help me understand what these problems are and how to troubleshoot them, I would be grateful.

**Comment by sjorge** [comment]
I accidentally introduce some issues  for the MOSZB-140 with older firmware versions. There is a lengthy thread here: https://github.com/Koenkk/zigbee2mqtt/issues/14277

The current dev branch has it mostly working again but there is one pending PR to squash one more bug.

Not sure on the SNZB-04 I don't own one of those.

**Reply by federicosantolini** [reply]
ok thanks I searched a lot and the thread #14277 never came up. I will follow it!

**Comment by sjorge** [comment]
All the extra fixes for the MOSZB-140 are now in the latest release from yesterday, it should work again without issue.
